assets/i18n/qqq.json

```diff
@@ -351,11 +351,22 @@
   "I18N_DONATE_PAGE_CONTENT_HEADING_1": "Heading of the first section of the donate page.",
   "I18N_DONATE_PAGE_CONTENT_HEADING_2": "Heading of the second section of the donate page.",
   "I18N_DONATE_PAGE_CONTENT_HEADING_3": "Heading of the third section of the donate page.",
+	"I18N_DONATE_PAGE_CONTENT_HEADING_4": "Heading of the fourth section of the donate page.",
```

Comment:
@amyyeung17 I think the indentation here and below is off?

Reply:
Whoops, I think Github interpreted the spacing as 8 instead of 2.

Reply:
Ah, make sure you use spaces and not tabs.

Reply:
@seanlip sorry, I'm a bit confused. I saw what you meant by the off indentation before yesterday's commit, but it looks fine now on my side. Am I perhaps misinterpreting something?

en.json
<img width="500" alt="Screen Shot 2024-01-11 at 12 42 58 PM" src="https://github.com/oppia/oppia/assets/28910543/d7582ec5-0123-4725-a608-dd55a55d78e8">

qqq.json
<img width="500" alt="Screen Shot 2024-01-11 at 12 37 23 PM" src="https://github.com/oppia/oppia/assets/28910543/534ae524-bee6-4493-af13-818631c13c92">

Reply:
For me, the character before "I18N_DONATE_PAGE_CONTENT_LEARNER_COUNTRY_1" seems to be a single "Tab" character. The characters before the other (correct) lines are 2 single-space characters.

Maybe try to read the ascii value of the file chars and see which it is? Sometimes text editors and IDEs mangle this.

Reply:
@seanlip That is correct. The dots are returning ascii values for 2 spaces, and the arrow is returning a tab value before "I18N_DONATE_PAGE_CONTENT_LEARNER_COUNTRY_1". I compared the ASCII values for the other lines and they all appear to be tabs as well. 

This is what the original file looks like from the develop branch viewed in the browser. 
![Screen Shot 2024-01-11 at 2 57 18 PM](https://github.com/oppia/oppia/assets/28910543/8acfc55b-2191-4dd8-8af7-fe2fed748a69)

Reply:
Done, normalized spacing for qqq.json.